schemas/configuration/v4/server_config.json.in

```diff
@@ -114,6 +114,7 @@
                     "authentication": {"enum": ["trace", "debug", "info", "warn", "error", "critical"], "maxItems": 1},
                     "database": {"enum": ["trace", "debug", "info", "warn", "error", "critical"], "maxItems": 1},
                     "delay_server": {"enum": ["trace", "debug", "info", "warn", "error", "critical"], "maxItems": 1},
+                    "genquery2": {"enum": ["trace", "debug", "info", "warn", "error", "critical"], "maxItems": 1},
```

Comment:
can't just do this here if you're bumping a version.   would need a whole new v5 directory.

Reply:
and can't do this here without bumping a version...